ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js

```diff
@@ -2,13 +2,13 @@ import React, { useContext } from 'react';
 import {
   FONT_WEIGHT,
```

Comment:
Hey @PrgrmrHarshShukla, we recently created enum version of the Typography helpers so you can now use that
```suggestion
  FontWeight,
```